src/app/doc_exporter.cpp

```diff
@@ -1180,10 +1180,13 @@ Doc* DocExporter::createEmptyTexture(const Samples& samples,
                                      base::task_token& token) const
 {
   ColorMode colorMode = ColorMode::INDEXED;
-  Palette* palette = nullptr;
+  Palette palette(0, 0);
   int maxColors = 256;
   gfx::ColorSpaceRef colorSpace;
   color_t transparentColor = 0;
+  const bool textureSupportsPalette =
+    format_supports_palette(m_textureFilename) ||
+    m_textureFilename == ""; // if m_textureFile == "" aseprite format is taken as default.
```

Comment:
Actually the comment is not correct, if the filename is empty, the `FileFormat::UNKNOWN` is used in `format_supports_palette()`.

My suggestion (add `(...)` between the expression, don't use `== ""` to compare empty strings, and fix the comment)
```suggestion
    (m_textureFilename.empty() || // This is the preview and we create the palette anyway
     format_supports_palette(m_textureFilename)); 
```